README.md

```diff
@@ -210,12 +244,6 @@ If you want to clone the repo for contributing or for running the example app yo
 git submodule update --init --recursive
 ```
 
-Or, if you have Carthage installed, you can use it to do the same thing:
-
-```
-carthage checkout
```

Comment:
This hasn't worked. I had submodules issues when I installed it with the carthage. I think it's better to remove it for now.